src/AsyncTCP.cpp

```diff
@@ -1017,7 +1018,6 @@ int8_t AsyncClient::_poll(tcp_pcb* pcb){
 }
 
 void AsyncClient::_dns_found(struct ip_addr *ipaddr){
-#if LWIP_IPV4 && LWIP_IPV6
     if(ipaddr && ipaddr->u_addr.ip4.addr){
         connect(IPAddress(ipaddr->u_addr.ip4.addr), _connect_port);
```

Comment:
Add
```
#else
    if (ipaddr && ipaddr->addr)
    {
        connect(IPAddress(ipaddr->addr), _connect_port);
#endif

Reply:
Thanks for taking the time to review the PR. I was mainly trying to raise awareness :)
I also noticed that another PR which tries to fix the same issue is also pending